certbot/storage.py

```diff
@@ -1144,7 +1144,8 @@ def save_successor(self, prior_version, new_cert,
             # If the previous privkey in this lineage has an existing gid or group mode > 0,
             # let's keep those.
             group_mode = stat.S_IMODE(os.stat(old_privkey).st_mode) & \
```

Comment:
nit: I think we should change the name of this variable and the comment above since it's no longer just the group mode.